content/en/docs/ops/common-problems/injection/index.md

```diff
@@ -141,23 +141,37 @@ Warning  FailedCreate  3m (x17 over 8m)  replicaset-controller  Error creating:
 caused by an empty `caBundle` in the webhook configuration.
 
 Verify the `caBundle` in the `mutatingwebhookconfiguration` matches the
-   root certificate mounted in the `istio-sidecar-injector` pod.
+   root certificate mounted in the `istio-sidecar-injector` or `istiod`(if you're using istiod) pod.
```

Comment:
Do we need to document this (and the other changes, below) two ways? Can't it just be switched to `istiod` now?
```suggestion
   root certificate mounted in the `iistiod` pod.
```

Reply:
Istiod is the only option in 1.6

Reply:
ok